stylesheets/components/Button/button.css

```diff
@@ -0,0 +1,153 @@
++/* Button Component Styling */
+.btn {
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+    cursor: pointer;
+    width: fit-content;
+    font-weight: 500;
+    padding: 0.75rem 1rem;
+    border-radius: 8px;
+    color: white;
+    border: 2px solid transparent;
+    background-color: var(--component-grey-01);
+    color: var(--component-grey-05);
+    transition: all 0.3s ease-in-out;
+  }
+
+  .btn:hover {
+    background-color: var(--component-grey-01);
+    color: #000;
+  }
+  /* Button Icons */
+  .btn .material-icons {
+    font-size: 1rem;
+    outline: 0px solid;
+  }
+
+  /* Primary Button */
+  .btn--primary {
+    background-color: var(--primary-color);
+  }
+  .btn--primary:hover {
+    color: var(--component-blue-04);
+    background-color: white;
```

Comment:
can you use variable of colour

Reply:
Hey Shubham! Thanks for that, I missed it from my end. Have corrected it. 
Thanks.